packages/button/src/ButtonBase.tsx

```diff
@@ -130,6 +130,7 @@ export const ButtonBase = styled.button<ButtonProps<'button'>>`
   cursor: pointer;
   display: inline-flex;
   font: ${primary.weight.bold} 1.6rem ${primary.family};
+  width: fit-content;
```

Comment:
Is there a case when inheriting is what we want?

Reply:
I don't see a use case currently, but if we do need one in the future we can always add a width prop that overrides this style. This is mainly so I don't have to add custom css to the button on the FE.

Reply:
If we encounter that situation, we can add a prop